ros-2.autobuild

```diff
@@ -9,10 +8,13 @@ import_package "slam/pointmatcher" do |pkg|
     pkg.depends_on "slam/nabo"
 end
 
-import_package "slam/slam3d_ros2" do |pkg|
-    pkg.depends_on "slam/slam3d"
-    pkg.depends_on "ament_cmake"
-end
+import_package "slam3d"
+move_package "slam3d", "slam"
+metapackage "slam/slam3d", "slam3d"
```

Comment:
In the ros2 world, "slam/slam3d" should not exist, when keeping the rename in package.xml generation (dependency_name_overrides.yml), defining a metapackage shouldn't be necessary